full/src/test/java/apoc/ApocSplitTest.java

```diff
@@ -0,0 +1,631 @@
+        "apoc.bolt.execute",
+        "apoc.config.list",
+        "apoc.config.map",
+        "apoc.redis.getSet",
+        "apoc.redis.get",
+        "apoc.redis.append",
+        "apoc.redis.incrby",
+        "apoc.redis.hdel",
+        "apoc.redis.hexists",
+        "apoc.redis.hget",
+        "apoc.redis.hincrby",
+        "apoc.redis.hgetall",
+        "apoc.redis.hset",
+        "apoc.redis.push",
+        "apoc.redis.pop",
+        "apoc.redis.lrange",
+        "apoc.redis.sadd",
+        "apoc.redis.scard",
+        "apoc.redis.spop",
+        "apoc.redis.smembers",
+        "apoc.redis.sunion",
+        "apoc.redis.zadd",
+        "apoc.redis.zcard",
+        "apoc.redis.zrangebyscore",
+        "apoc.redis.zrem",
+        "apoc.redis.eval",
+        "apoc.redis.copy",
+        "apoc.redis.exists",
+        "apoc.redis.pexpire",
+        "apoc.redis.persist",
+        "apoc.redis.pttl",
+        "apoc.redis.info",
+        "apoc.redis.configGet",
+        "apoc.redis.configSet",
+        "apoc.systemdb.export.metadata",
+        "apoc.systemdb.graph",
+        "apoc.systemdb.execute",
+        "apoc.algo.aStarWithPoint",
+        "apoc.get.nodes",
+        "apoc.get.rels",
+        "apoc.cypher.runFile",
+        "apoc.cypher.runFiles",
+        "apoc.cypher.runSchemaFile",
+        "apoc.cypher.runSchemaFiles",
+        "apoc.cypher.parallel",
+        "apoc.cypher.mapParallel",
+        "apoc.cypher.mapParallel2",
+        "apoc.cypher.parallel2",
+        "apoc.gephi.add",
+        "apoc.mongo.aggregate",
+        "apoc.mongo.count",
+        "apoc.mongo.find",
+        "apoc.mongo.insert",
+        "apoc.mongo.update",
+        "apoc.mongo.delete",
+        "apoc.mongodb.get.byObjectId",
+        //"apoc.load.xls",
+        "apoc.load.csv",
+        "apoc.load.csvParams",
+        "apoc.load.ldap",
+        "apoc.load.driver",
+        "apoc.load.jdbc",
+        "apoc.load.jdbcUpdate",
+        "apoc.load.htmlPlainText",
+        "apoc.load.html",
+        "apoc.load.directory.async.add",
+        "apoc.load.directory.async.remove",
+        "apoc.load.directory.async.removeAll",
+        "apoc.load.directory.async.list",
+        "apoc.load.directory",
+        "apoc.dv.catalog.add",
+        "apoc.dv.catalog.remove",
+        "apoc.dv.catalog.list",
+        "apoc.dv.query",
+        "apoc.dv.queryAndLink",
+        "apoc.model.jdbc",
+        "apoc.generate.ba",
+        "apoc.generate.ws",
+        "apoc.generate.er",
+        "apoc.generate.complete",
+        "apoc.generate.simple",
+        "apoc.log.error",
+        "apoc.log.warn",
+        "apoc.log.info",
+        "apoc.log.debug",
+        "apoc.es.stats",
+        "apoc.es.get",
+        "apoc.es.query",
+        "apoc.es.getRaw",
+        "apoc.es.postRaw",
+        "apoc.es.post",
+        "apoc.es.put",
+//        "apoc.export.xls.all",
+//        "apoc.export.xls.data",
+//        "apoc.export.xls.graph",
+//        "apoc.export.xls.query",
```

Comment:
I think there's something broken cause this should be in full, and it's packaged in the jar but it's not available in the procedures. Way to reproduce:

```sh
docker run --restart always --publish=7474:7474 --publish=7687:7687 --env=NEO4J_ACCEPT_LICENSE_AGREEMENT=yes --env NEO4JLABS_PLUGINS='["apoc"]' neo4j:4.4.9-enterprise
```

```
// In cypher-shell

neo4j@neo4j> RETURN apoc.version();
+----------------+
| apoc.version() |
+----------------+
| "4.4.0.7"      |
+----------------+

1 row
ready to start consuming query after 75 ms, results consumed after another 2 ms
neo4j@neo4j> CALL apoc.export.xls.all
             ;
There is no procedure with the name `apoc.export.xls.all` registered for this database instance. Please ensure you've spelled the procedure name correctly and that the procedure is properly deployed.
```

Reply:
Oki that should be fine. Maybe we could add a todo to make sure it doesn't get forgotten about?